ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -343,52 +370,129 @@ def install(cls, mh):
             setup_dns=False, extra_args=['--no-ntp']
         )
 
-    def test_renew_expired_cert_replica(self):
+    @pytest.fixture
+    def expire_certs(self):
+        # move system date to expire certs
+        for host in self.master, self.replicas[0]:
+            tasks.move_date(host, 'stop', '+3years+1days')
+
+        yield
+
+        # move date back on replica and master
+        for host in self.master, self.replicas[0]:
+            tasks.move_date(host, 'start', '-3years-1days')
+
+    def test_renew_expired_cert_replica(self, expire_certs):
         """Test renewal of certificates on replica with ipa-cert-fix
 
         This is to check that ipa-cert-fix renews the certificates
         on replica
 
         related: https://pagure.io/freeipa/issue/7885
         """
-        tasks.move_date(self.master, 'stop', '+3years+1days')
-
         # wait for cert expiry
         check_status(self.master, 8, "CA_UNREACHABLE")
 
         self.master.run_command(['ipa-cert-fix', '-v'], stdin_text='yes\n')
 
         check_status(self.master, 9, "MONITORING")
 
-        # move system date to expire cert on replica
-        tasks.move_date(self.replicas[0], 'stop', '+3years+1days')
-
-        # RA agent cert will be expired and in CA_UNREACHABLE state
-        check_status(self.replicas[0], 1, "CA_UNREACHABLE")
-
-        # renew RA agent cert
-        self.replicas[0].run_command(
-            ['ipa-cert-fix', '-v'], stdin_text='yes\n'
+        # replica operations
+        # 'Server-Cert cert-pki-ca' cert will be in CA_UNREACHABLE state
+        cmd = self.replicas[0].run_command(
+            ['getcert', 'list',
+             '-d', paths.PKI_TOMCAT_ALIAS_DIR,
+             '-n', 'Server-Cert cert-pki-ca']
         )
+        req_id = get_certmonger_fs_id(cmd.stdout_text)
+        tasks.wait_for_certmonger_status(
+            self.replicas[0], ('CA_UNREACHABLE'), req_id, timeout=600
+        )
+        # get initial expiry date to compare later with renewed cert
+        self.replicas[0].run_command([
+            'certutil', '-L', '-a',
+            '-d', paths.PKI_TOMCAT_ALIAS_DIR,
+            '-n', '"Server-Cert cert-pki-ca"',
```

Comment:
No need for both single and double quotes, `'-n', 'Server-Cert cert-pki-ca',` should be enough